apiary.apib

```diff
@@ -972,10 +972,309 @@ Outputs the list of peers that were removed.
 ## query
 
 # Group diag
+Generate diagnostic reports.
 
-## net
+This command can't be called directly.
 
-## sys
+## net [GET /diag/net{?vis}]
+Sends out a message to each node in the network recursively
+requesting a listing of data about them including number of
+connected peers and latencies between them.
+
+The given timeout will be decremented 2s at every network hop,
+ensuring peers try to return their diagnostics before the initiator's
+timeout. If the timeout is too small, some peers may not be reached.
+30s and 60s are reasonable timeout values, though network vary.
+The default timeout is 20 seconds.
+
++ Parameters
+    + vis (enum[string], optional)
```

Comment:
No, it is global. I'll need to add a note about that, but it isn't actually relevant to this PR.